nodes/persistence-node/package.json

```diff
@@ -38,7 +38,8 @@
     "custom-env": "^2.0.1",
     "ethers": "^5.5.2",
     "express": "^4.17.2",
-    "ipfs-core": "^0.12.2",
```

Comment:
Let's start pinning dependencies, since we have that practice in other projects.
Just remove "^" everywhere

Reply:
Should we just commit `yarn.lock` file instead?

Reply:
we do that already, but yarn.lock sometimes gets buggy and we regenerate them, this would be an added layer of protection. Not ideal though

Reply:
yes the lock file is not enough, please pin all versions :pray:

Reply:
Fixed.